### PR TITLE
Seed data refactor

### PR DESCRIPTION
**Pull request by BenAndrew-Kainos**

### CODE REVIEW

File: ui_automation_tests/conftest.py [hunk 1]
```diff
@@ -4,7 +4,7 @@
 from pytest_bdd import given, when, then, parsers
 from selenium.webdriver.common.by import By
 
-from fixtures.core import context, driver, invalid_username, exporter_sso_login_info, s3_key  # noqa
+from fixtures.core import context, driver, invalid_username, exporter_sso_login_info, s3_key, seed_data_config  # noqa
```
Comment (a-bek) [inline ui_automation_tests/conftest.py]: unused import?

File: ui_automation_tests/fixtures/core.py [hunk 2]
```diff
@@ -67,3 +67,10 @@ def s3_key(request):
 @fixture(scope="session")
 def invalid_username(request):
     return "invalid@mail.com"
+
+
+@fixture(scope="session")
+def seed_data_config(request):
+    exporter_sso_email = env('TEST_EXPORTER_SSO_EMAIL')
```
Comment (phlptt) [inline ui_automation_tests/fixtures/core.py]: can this use the test export sso email from fixtures

File: ui_automation_tests/fixtures/core.py [hunk 3]
```diff
@@ -67,3 +67,10 @@ def s3_key(request):
 @fixture(scope="session")
 def invalid_username(request):
     return "invalid@mail.com"
+
+
+@fixture(scope="session")
+def seed_data_config(request):
```
Comment (mh-hmcts) [inline ui_automation_tests/fixtures/core.py]: add the url into this as well?

File: ui_automation_tests/conftest.py (hunk 1 above)
Comment (BenAndrew-Kainos) [reply ui_automation_tests/conftest.py]: fixtures are all imported in conftest.py and passed down from this. pyCharm thinks it's unused. Talk to Nick for furthe info

**Pull request merged**